receiver/mysqlreceiver/metadata.yaml

```diff
@@ -283,6 +283,15 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.connection.count:
+    enabled: true
```

Comment:
The comment is not to block this PR, but to ensure that someone starts thinking about this and makes a proposal.

Reply:
So far, my heuristics for disabling metrics by default have been for things like:
- If it creates high cardinality of resources and/or data points 
- If there is a notable expected performance impact
- If there is unusual configuration required (on monitored system or collector)

In general, I think of metrics as otherwise being pretty cheap, but I agree we should probably establish some kind of limitations based on how valuable a metric is vs others.

Reply:
Should it go as enabled, or should I disable it and we can enable them in separate PR eventually?

Reply:
@djaglowski @bogdandrutu
Disabled metric by default and rebased the branch